frontend/src/concepts/k8s/NameDescriptionField.tsx

```diff
@@ -87,10 +129,34 @@ const NameDescriptionField: React.FC<NameDescriptionFieldProps> = ({
               name={`resource-${nameFieldId}`}
               value={data.k8sName ?? k8sName}
               onChange={(k8sName) => {
+                validateResourceName(k8sName);
                 setData({ ...data, k8sName });
               }}
-              validated={!isValidK8sName(data.k8sName) ? 'error' : undefined}
+              aria-describedby={Object.keys(validateK8sName)
+                .filter((item: string) => validateK8sName[item] !== 'success')
+                .join(' ')}
+              aria-invalid={ruleCharacterTypes === 'error' || ruleLength === 'error'}
             />
+
+            <HelperText component="ul">
+              <HelperTextItem
+                component="li"
+                id="ruleLength"
+                isDynamic
+                variant={ruleLength as HelperTextItemProps['variant']}
+              >
+                {`Cannot exceed 30 characters. Otherwise, the excess will be truncated.`}
+              </HelperTextItem>
+              <HelperTextItem
+                component="li"
+                id="ruleCharacterTypes"
+                isDynamic
+                variant={ruleCharacterTypes as HelperTextItemProps['variant']}
+              >
+                {`Must consist of lower case alphanumeric characters or '-', and must start and end
+                with an alphanumeric character.`}
```

Comment:
I thought that starting with a number caused the notebook to fail to start? Shouldn't this follow RF 1035:

- contain only lowercase alphanumeric characters or '-'
- start with an alphabetic character
- end with an alphanumeric character